### PR TITLE
updated changelog 0.26 (larray-editor)

### PR DESCRIPTION
**Pull request by alixdamman**
This PR can wait for next updates from closed PR in larray-editor. 
Need to be merged before release of version 0.26

### CODE REVIEW

**Comment by gdementen** [comment]
Here is a comment I typed yesterday but just did not send:
hmmm. I realize now that I am unsure the inversion is an improvement overall. This is really depends on the data and I don't know which is more prevalent.

When you have negative and positive values, or, more generally when lower values are bad, it feels natural to have red for negative, blue for positive, but when that is not the case, having "red" for larger values which are "hotter" than low "colder" values seems more sensible.

I am totally unsure what to do now. Providing a dropdown with a choice between multiple gradients, like Excel would be best but that would take a bit of time if we want to have a preview of the gradient in the dropdown.

**Comment by gdementen** [comment]
But since then I worked on it in the train and I have a "gradient" combobox almost working...

File: doc/source/changes/version_0_26.rst.inc [hunk 1]
```diff
@@ -6,12 +6,15 @@
   results, but inside a function local_arrays will return only arrays local to the function, while global_arrays will
   return only arrays defined globally.
 
+* indicator `*` is appended to the title window when unsaved changes are detected in viewer (closes :editor_issue:`21`).
```
Comment (gdementen) [inline doc/source/changes/version_0_26.rst.inc]: * replace "indicator `*`" by either "a `*` indicator" or even "a `*` symbol" (my personal preference)
* s/title window/window title/
* s/in viewer/in the viewer/

File: doc/source/changes/version_0_26.rst.inc [hunk 2]
```diff
@@ -6,12 +6,15 @@
...
 
 Miscellaneous improvements
 --------------------------
 
 * view() and edit() without argument now display global arrays in addition to local ones (closes :editor_issue:`54`).
 
+* inverted background colors in viewer (red for low values and blue for high values). Closes :editor_issue:`18`.
```
Comment (gdementen) [inline doc/source/changes/version_0_26.rst.inc]: s/in viewer/in the viewer/

File: doc/source/changes/version_0_26.rst.inc [hunk 3]
```diff
@@ -47,4 +50,4 @@ Miscellaneous improvements
 Fixes
 -----
 
-* fixed something (closes :issue:`1`).
+* fixed background color in viewer when calling the function `compare` (closes :editor_issue:`66`).
```
Comment (gdementen) [inline doc/source/changes/version_0_26.rst.inc]: s/in viewer when.../when using filters in the `compare()` dialog/

**Comment by alixdamman** [comment]
Once again, you cannot see my two last added commits... Strange.

**Comment by alixdamman** [comment]
Now you see.

File: doc/source/changes/version_0_26.rst.inc [hunk 4]
```diff
@@ -321,3 +328,13 @@ Fixes
...
+
+* fixed background color in the viewer when using filters in the `compare()` dialog (closes :editor_issue:`66`)
+
+* fixed autoresize of columns by double clicking between column headers (closes :editor_issue:`43`)
+
+* fixed representing a 0D array (scalar) in the viewer (closes :editor_issue:`71`)
+
+* fixed viewer not displaying an error message when saving or loading a file failed (closes :editor_issue:`75`)
+
+* fixed viewer showing array with empty axes names and labels when starting a new session (closes :editor_issue:`84`)
```
Comment (gdementen) [inline doc/source/changes/version_0_26.rst.inc]: * isn't this the same issue than the 0D array issue above (71)?
* did those two issues actually exist in 0.25.2? I don't remember. The issues we introduced in 0.26 (before fixing them), should not be mentioned in the changelog.

Comment (alixdamman) [reply doc/source/changes/version_0_26.rst.inc]: issue 84 and 71 are a bit different. 
Issue 71 concerns array of size 1 but no axes.
Issue 84 concerns "empty" array, i.e. of size 0 but with potentially fake axes ( try arr = LArray([]) )

**Comment by gdementen** [comment]
let's merge this :)

**Pull request merged**